java/src/main/java/com/saucelabs/simplesauce/Options.java

```diff
@@ -0,0 +1,269 @@
+            return this.value.toString();
+        }
+    }
+
+    public enum JobVisibility {
+        PUBLIC("public"),
+        PUBLIC_RESTRICTED("public restricted"),
+        SHARE("share"),
+        TEAM("team"),
+        PRIVATE("private");
+
+        @Getter private String value;
+
+        public static Set keys() {
+            return JobVisibilityLookup.lookup.keySet();
+        }
+
+        JobVisibility(String value) {
+            this.value = value;
+            JobVisibilityLookup.lookup.put(value, this.name());
+        }
+
+        public static String fromString(String value) {
+            return JobVisibilityLookup.lookup.get(value);
+        }
+
+        public String toString() {
+            return this.value.toString();
+        }
+    }
+
+    public enum Browser {
+        CHROME("chrome"),
+        INTERNET_EXPLORER("internet explorer"),
+        EDGE("MicrosoftEdge"),
+        SAFARI("safari"),
+        FIREFOX("firefox");
+
+        @Getter private final String value;
+
+        public static Set keys() {
+            return BrowserLookup.lookup.keySet();
+        }
+
+        Browser(String value) {
+            this.value = value;
+            BrowserLookup.lookup.put(value, this.name());
+        }
+
+        public static String fromString(String value) {
+            return BrowserLookup.lookup.get(value);
+        }
+
+        public String toString() {
+            return this.value;
+        }
+    }
+
+    public enum Timeouts {
+        IMPLICIT("implicit"),
+        PAGE_LOAD("pageLoad"),
+        SCRIPT("script");
+
+        @Getter private final String value;
+
+        public static Set keys() {
+            return TimeoutsLookup.lookup.keySet();
+        }
+
+        Timeouts(String value) {
+            this.value = value;
+            TimeoutsLookup.lookup.put(value, this.name());
+        }
+
+        public static String fromString(String value) {
+            return TimeoutsLookup.lookup.get(value);
+        }
+
+        public String toString() {
+            return this.value;
+        }
+    }
+
+    public enum Platform {
+        WINDOWS_10("Windows 10"),
+        WINDOWS_8_1("Windows 8.1"),
+        WINDOWS_8("Windows 8"),
+        MAC_MOJAVE("macOS 10.14"),
+        MAC_HIGH_SIERRA("macOS 10.13"),
+        MAC_SIERRA("macOS 10.12"),
+        MAC_EL_CAPITAN("OS X 10.11"),
+        MAC_YOSEMITE("OS X 10.10");
+
+        @Getter
+        private final String value;
+
+        public static Set keys() {
+            return PlatformLookup.lookup.keySet();
+        }
+
+        Platform(String value) {
+            this.value = value;
+            PlatformLookup.lookup.put(value, this.name());
+        }
+
+        public static String fromString(String value) {
+            return PlatformLookup.lookup.get(value);
+        }
+
+        public String toString() {
+            return this.value;
+        }
+    }
+
+    public static final List<String> primaryEnum = List.of(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/SauceLabs/simple_sauce/pullRequest?prid=4758011)